i18n/ar.json

```diff
@@ -8,24 +8,25 @@
 			"أحمد",
 			"الهميان",
 			"روخو",
-			"محمد أحمد عبد الفتاح"
+		"محمد أحمد عبد الفتاح",
+			"أحمد غربية Ahmad Gharbeia <ahmad@arabdigitalexpression.org>"
 		]
 	},
-	"smw-desc": "تقرّب الويكي إلى كلٍّ من النّظم الحاسوبية  ''و''البشر ([https://www.semantic-mediawiki.org/wiki/Help:User_manual دليل المستخدم])",
+	"smw-desc": "تقرّب الويكي إلى كلٍّ من النّظم الحاسوبية  ''و'' البشر ([https://www.semantic-mediawiki.org/wiki/Help:User_manual دليل المستخدم])",
 	"smw_viewasrdf": "تلقيمة RDF",
 	"smw_finallistconjunct": " و",
 	"smw_factbox_head": "حقائق عن $1",
 	"smw_isspecprop": "هذه خصيصة مميَّزة في هذه الويكي.",
 	"smw_concept_description": "وصف المفهوم \"$1\"",
-	"smw_no_concept_namespace": "المفاهيم لا يمكن تعريفها إلا في الصفحات في فضاء التسمية Concept",
+	"smw_no_concept_namespace": "المفاهيم لا يمكن تعريفها إلا في الصفحات في فضاء التسمية \"مفهوم\"",
 	"smw_multiple_concepts": "كل صفحة مفهوم لا يمكن أن تضمّ أكثر من تعريف مفهوم واحد.",
-	"smw_concept_cache_miss": "المبدأ \"$1\" لا يمكن استخدامه حاليا إذ أن كيفية ضبط الويكي تقضي بحسابه خارج الخطّ. إذا لم تنته المشكلة بعد برهة، فسَل إداري الموقع ليتيح هذا المبدأ.",
+	"smw_concept_cache_miss": "المبدأ \"$1\" لا يمكن استخدامه حاليا إذ أن كيفية ضبط الويكي تقضي بتعطيلها. إذا لم تنته المشكلة بعد برهة، فسَل إداري الموقع ليتيح هذا المبدأ.",
 	"smw_noinvannot": "لا يمكن تعيين قيم لخصائص معكوسة.",
 	"version-semantic": "ملحقات دلالية",
 	"smw_baduri": "المسارات في الصيغة \"$1\" غير مسموح بها هنا.",
-	"smw_csv_link": "سي في إس",
-	"smw_json_link": "جسون",
-	"smw_rdf_link": "آر دي إف",
+	"smw_csv_link": "CVS",
+	"smw_json_link": "JSON",
+	"smw_rdf_link": "RDF",
```

Comment:
I'm not sure this can count as "untranslation". It could be "untransliteration". The Arabic strings were transliterations of the names of the English letters. There's not much information here.
The reason I cite in the commit note as justification for this change is that the Latin-script acronyms, in my claim, are immediately recognisable for what they stand for. The same could not be said about their Arabic transliterations, which merely add a second layer of "resolution". In a predominantly visual medium like the web I find this not useful.